src/integrationtests/java/com/aws/greengrass/integrationtests/lifecyclemanager/KernelTest.java

```diff
@@ -245,7 +245,7 @@ void GIVEN_service_startup_always_fail_WHEN_kernel_launches_THEN_service_go_brok
                 serviceBroken.countDown();
             }
         });
-        assertTrue(serviceBroken.await(TIMEOUT, TimeUnit.SECONDS));
+        assertTrue(serviceBroken.await(60, TimeUnit.SECONDS));
```

Comment:
why increase 45s -> 60s?  was the test flaky?

Reply:
Yes, timing out on windows.